packages/atlas/src/views/viewer/VideoView/VideoView.tsx

```diff
@@ -139,7 +136,7 @@ export const VideoView: FC = () => {
   const categoryId = video?.category?.id
   const numberOfLikes = video?.reactions.filter(({ reaction }) => reaction === 'LIKE').length
   const numberOfDislikes = video?.reactions.filter(({ reaction }) => reaction === 'UNLIKE').length
-  const videoIsBlocked = !!(id && filteredVideoIds.includes(id))
+  const videoNotAvailable = !!(id && filteredVideoIds.includes(id)) || (!loading && !video)
```

Comment:
What's the point of `!!(id && filteredVideoIds.includes(id))`? If the video is blocked, the second clause will still capture that?